circle.yml

```diff
@@ -228,36 +228,38 @@ jobs:
    centos6:
      working_directory: ~/universal-ctags
      docker:
-       - image: centos:6
+       - image: centos:6.10
      steps:
        - run:
            name: Rearrange CentOS-Base.repo file
            command: |
-             sed -i 's|#baseurl=|baseurl=|g' /etc/yum.repos.d/CentOS-Base.repo
+             sed -i 's|#baseurl=|baseurl=|g'       /etc/yum.repos.d/CentOS-Base.repo
```

Comment:
The changes about whitespaces are nothing to do with the original issue that you wanted to solve.
So I woud like you to split the pull request into two if you want to change the whitespaces; one is for updating whitespaces and another is for fixing the original issue. In that case, I would like to use "circuleci,cosmetic: ..." as the prefix for the comment header for the former commit.
See 2a0a14105b2ac169f264f0a609883392223a4842 aa6af6384d758dfb55e47079dbece38275d0733cc nd as examples for the former commit.